lib/YaST/Firewall/StartUpPage.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class introduces all accessing methods for YaST module
+# System Settings Page.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::Firewall::StartUpPage;
```

Comment:
this should be encapsulated in ServiceConfigurationPage, as we will be reusing this outside /Firewall/ in other YaST modules.

Reply:
The module name is consistent with yast UI, so you will map it 1-to-1 more easier if you need do more development.
From outside we should call function $testapi::distri->get_firewall()->start_firewall() etc.  this should not impact end user's usage.